nimare/tests/test_cbma_ale.py

```diff
@@ -66,15 +70,34 @@ def test_ale(testdata1):
     """
     Smoke test for ALE
     """
-    ale_meta = ale.ALE(n_iters=5, n_cores=1)
+    ale_meta = ale.ALE()
     ale_meta.fit(testdata1)
+    corr = FWECorrector(method='permutation')
+    result_corr = corr.transform(ale_meta.results, voxel_thresh=0.001,
+                                 n_iters=5, n_cores=1)
```

Comment:
I think this is what standard usage would look like. WDYT?

Reply:
I like it, with two possible modifications:

1. I think maybe `.fit` should return the `MetaResult` object by default (though it should also set it internally).
2. The arguments to `transform` should probably be passed to the `FWECorrector` at initialization.

There's also an alternative pattern we could consider, which is to make the correction classes functors (i.e., classes that are called like functions). That would allow us to do something like:

```python
ale_meta = ale.ALE()
result = ale_meta.fit(cbma_testdata1)
result_corr = fwe_correct(result, method='permutation', voxel_thresh=0.001, n_iters=5, n_cores=1)
```

Internally, this would actually be a pretty small change from the present implementation. `fwe_correct` would still be defined as a class, and we would have the same class hierarchy. The main difference is we would direct `__call__` to `transform` (or just replace the latter with the former).

It would make for a slightly cleaner API; the main drawback is the loss of consistency with the APIs of the other major class hierarchies. I don't have a strong preference either way (might be worth polling a few other people).

Reply:
> 1. I think maybe .fit should return the MetaResult object by default (though it should also set it internally).
That's easy enough. I'll do that.

> 2. The arguments to transform should probably be passed to the FWECorrector at initialization.
I think I've got some questions in another comment, but definitely working on this.

I'm fine with holding off on converting the Correctors to functors until a later date, depending on how people feel about using them as they currently are. Once this is merged and the Correctors are all working, I can raise an issue to get folks' opinions.

Reply:
I spent some time today looking into functors to maybe solve #73 and #74, but it looks like they still need to be initialized (i.e., `func_inst = functor(); func_inst('call')` instead of `functor('call')`). I could be missing something though.